client/state/notices/reducer.js

```diff
@@ -31,6 +32,16 @@ export const items = createReducer( {}, {
 
 		return omit( state, noticeId );
 	},
+	[ NOTICE_CLEAR ]: ( state ) => {
+		return reduce( state, ( memo, notice, noticeId ) => {
```

Comment:
I think you're right. I thought about adding a "tag" or "tags" attribute to the notices, that way we can delete notices by tag. Do you think this is a good solution. (All Site Settings Notices will be tagged by `site-settings`, so we could trigger a `clearNotices('site-settings')`. Thoughts on this?